cmd/reposervercontroller/main.go

```diff
@@ -40,7 +39,10 @@ import (
 	"github.com/kanisterio/kanister/pkg/log"
 	"github.com/kanisterio/kanister/pkg/resource"
 	"github.com/kanisterio/kanister/pkg/validatingwebhook"
+	//nolint:gci
```

Comment:
what is the error that we get if we don't ignore this here?

Reply:
when I run `golangci-lint run` it automatically removes `//+kubebuilder:scaffold:imports` which we don't want to.